dbt_copilot_helper/commands/codebase.py

```diff
@@ -96,8 +96,9 @@ def prepare():
 def list(app, with_images):
     """List available codebases for the application."""
     application = load_application_or_abort(app)
-    ssm_client = boto3.client("ssm")
-    ecr_client = boto3.client("ecr")
+    session = get_aws_session_or_abort()
```

Comment:
This works, but also we're calling `get_aws_session_or_abort()` inside the `load_application_or_abort(app)` call. Not a major thing, but many calls...